server/lib/mappings/result-data-sample.json

```diff
@@ -15,16 +15,6 @@
         "@generated-by": {
             "type": "keyword"
         },
-        "authorization": {
-            "properties": {
-                "owner": {
-                    "type": "keyword"
-                },
-                "access": {
-                    "type": "keyword"
-                }
-            }
-        },
```

Comment:
Why is it good/better to add these programmatically, after the fact than to do it statically in the various files?  And, why doesn't that logic apply to the other mappings, et al. defined in these files?

Reply:
The idea (proposed by Pete in an earlier review cycle) is that this is common to all our Elasticsearch documents; e.g., if we support customized document format plugins we'll still want these to support our access control. Adding them after the fact guarantees they're consistent across all documents, and never omitted.

There really isn't anything else that's common across them all, although a few items are common across several, or similar. This seemed like a good convention to me.